src/components/OnboardingPage.tsx

```
@@ -41,6 +41,32 @@ export default function OnboardingPage() {
+      const hashParams = new URLSearchParams(window.location.hash.substring(1))
+      const accessToken = hashParams.get('access_token')
+      
+      if (accessToken) {
+        // Decode JWT to get email
+        try {
+          const base64Url = accessToken.split('.')[1]
+          const base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/')
+          const jsonPayload = decodeURIComponent(
+            atob(base64)
+              .split('')
+              .map(c => '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2))
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)